js/scripts.js

```diff
@@ -0,0 +1,5 @@
+var buttonElements = document.getElementsByClassName("button");
+	for (var i = 0; i < buttonElements.length; i++) {
```

Comment:
move `buttonElements.length` to new variable, if you let it in that case, this value will be calc on every loop turn